test/integration/utility.cc

```diff
@@ -221,27 +223,31 @@ IntegrationUtil::makeSingleRequest(const Network::Address::InstanceConstSharedPt
     // Docker only works with loopback v6 address.
     local_address = std::make_shared<Network::Address::Ipv6Instance>("::1");
   }
-  Network::ClientConnectionPtr connection =
-      Quic::createQuicNetworkConnection(*persistent_info, *dispatcher, addr, local_address);
+  Network::ClientConnectionPtr connection = Quic::createQuicNetworkConnection(
+      *persistent_info, *dispatcher, addr, local_address, quic_stat_names, scope);
   connection->addConnectionCallbacks(connection_callbacks);
   Http::CodecClientProd client(type, std::move(connection), host_description, *dispatcher, random);
   // Quic connection needs to finish handshake.
   dispatcher->run(Event::Dispatcher::RunType::Block);
   return sendRequestAndWaitForResponse(*dispatcher, method, url, body, host, content_type, client);
 #else
   ASSERT(false, "running a QUIC integration test without compiling QUIC");
+  (void)quic_stat_names;
```

Comment:
use `UNREFERENCED_PARAMETER(quic_stat_names)` in ./source/common/common/macros.h